proposals/Exceptions.md

```diff
@@ -240,19 +234,21 @@ If the `kind` is `Exception`:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type  | `varuint32` | exception index to table in the *exception type section* |
+| tag   | varuint32 | index into exception section |
```

Comment:
This still looks wrong. Imports are disjoint from locally defined entities, they neither can nor should reference the exception section.